src/stylesheets/custom.scss

```diff
@@ -16,6 +16,11 @@
   font-weight: bold;
 }
 
+// Without this, <Card> components appear to render the <li> with a bullet point
+.usa-card {
+  display: block
```

Comment:
Probably better to replace this with `list-style-type: none`, so it doesn't interfere with any other layout or styling rules, and removes the need for the comment.

Reply:
6efe49bc